src/WebApp/flask/templates/aztkIns.html

```diff
@@ -3,10 +3,11 @@
 <p>
 <a href="https://{{assets}}.scm.azurewebsites.net/DebugConsole">Function App Console </a>
 </p>
-<p> <b>Cluster Information</b> <br>
-    {{cluster_info}}
-</p>
 <p>
 <a href="https://github.com/Azure/AI-PredictiveMaintenance/blob/master/docs/User%20Guides/AZTK%20Instructions.md">Create Cluster Using AZTK and spin up Jupyter Notebooks </a> 
 </p>
+<p> <b>Cluster Information: </b> <br>
```

Comment:
indentation is off.

Reply:
never ever use &lt;br&gt; it's legacy non-semantic tag that messes up the layout.